src/timer.ts

```diff
@@ -37,9 +37,8 @@ export class Timer {
 		this.pomosSinceStart = 0;
 		this.cyclesSinceLastAutoStop = 0;
 
-		if (this.settings.whiteNoise === true) {
```

Comment:
If you're initializing the white noise player on load every time, you don't need this conditional.

Reply:
Actualy I removed it.

![Uploading image.png…]()